python_modules/libraries/dagster-looker/dagster_looker/api/components/looker_component.py

```diff
@@ -229,4 +256,14 @@ def build_defs_from_state(
 
         specs = self._load_asset_specs(instance_data)
 
+        # If PDT builds are configured, build corresponding executable asset definitions
+        if self.pdt_builds:
+            requests_for_looker = [
+                RequestStartPdtBuild(**pdt_config.model_dump()) for pdt_config in self.pdt_builds
+            ]
```

Comment:
this should no longer be necessary once self.pdt_builds is just a list of RequestStartPdtBuilds